### PR TITLE
fix: change place of camera temperature and add coloration

### PR DESCRIPTION
**Pull request by alexispinson**
- Move camera temperature under image source
-  set the color red over 80% and the color orange over 30% by using macro (as etienne asked for a function, i noticed there were not that many repetition of a block, so i decided to keep the macro, tell me if you prefer to have functions)

### CODE REVIEW

File: Holovibes/sources/thread/information_worker.cc [hunk 1]
```diff
@@ -302,14 +305,18 @@ void InformationWorker::display_gui_information()
...
         to_display << "<tr><td>" << indication_type_to_string_.at(key) << "</td><td>" << *value << "</td></tr>";
+        if (key == IndicationType::IMG_SOURCE)
+        {
+            if (fps_map.contains(IntType::TEMPERATURE) && temperature_ != 0)
+            {
+                to_display << "<tr><td>" << fps_type_to_string_.at(IntType::TEMPERATURE) << "</td><td>" << temperature_
+                           << "°C</td></tr>";
+            }
+        }
+    }
```
Comment (Baschtien) [inline Holovibes/sources/thread/information_worker.cc:319]: Can merge the two ifs

**Review by Baschtien — COMMENTED** [review]
Fine but the coloration macros are used in 2 formats: percentage (90, 70) and decimal percentage(0.4, 0.75)
Maybe use only one ?

File: Holovibes/sources/thread/information_worker.cc [hunk 2]
```diff
@@ -216,9 +219,9 @@ std::string gpu_load()
     // Print GPU load
     auto load = gpuLoad.gpu;
     ss << "<td style=\"color:";
-    if (load < 80)
+    if (load < ORANGE_COLORATION_PERCENTAGE)
         ss << "white";
-    else if (load < 90)
+    else if (load < RED_COLORATION_PERCENTAGE)
```
Comment (Hpn4) [inline Holovibes/sources/thread/information_worker.cc:224]: Maybe create a small function to avoid duplicate

**Review by ArthurCourselle — APPROVED** [review]
Except @Baschtien comment I agree with the changes.

**Review by TitouanGragnic — APPROVED** [review]

**Comment by alexispinson** [comment]
Approve for me

**Pull request merged**